src/System.Device.Gpio/System.Device.Gpio.csproj

```diff
@@ -25,17 +21,8 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsLinux)' == 'true'">
-    <!--Excluding Windows implementations-->
-    <Compile Remove="**\*.Windows.cs" />
-    <Compile Remove="Interop\Windows\**\*.cs" />
```

Comment:
This does mean that now the Linux implementation will depend on some winmd files, obviously not at runtime but just wanted to note that. I don't expect this to be a problem, but there are some runtimes that will fail to load an assembly when one of the dependencies is not present, that said I think that only applies to AOT so we are most likely safe here.

Reply:
What is AOT? 

Apparently, it runs on all existing platforms (at least all builds are green). For Windows and Linux I tested that the exceptions only pop up when the corresponding (in this case: the wrong) classes are instantiated. But that's the behavior we expect.

Reply:
AOT is ahead-of-time so basically it is when IL is compiled down to native from compile time. There are several frameworks that work like that, like UWP or Xamarin. Again, .NET Core is not like that so I think we are safe so we probably don't need to do anything special here.